### PR TITLE
making travis build master / develop

### PR DESCRIPTION
**Pull request by ateliercw**
adding an only build on push for develop / master to the travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ env:
   global:
     - LC_CTYPE=en_US.UTF-8
 
+branches:
```
Comment (dostrander) [inline .travis.yml]: #? why only develop and master, dont we want the branches to build to know whether or not to merge them in?

Comment (ateliercw) [inline .travis.yml]: from my understanding of the docs, that means any pushes to those branches, and any pull requests onto those branches

**Comment by dostrander** [comment]
@ateliercw I can't specifically find it but i think you are right, i was just expecting that was not the case. Merge when built!

**Comment by ateliercw** [comment]
adding shields to address #16

**Pull request merged**